sdk/src/connections/target/adsd3500_sensor.h

```diff
@@ -187,8 +187,8 @@ class Adsd3500Sensor : public aditof::DepthSensorInterface,
              {"depth", 1024, 1024},
              {"conf", 1024, 1024},
              {"embedded_header", 1, 128}},
-            1024,
-            4096,
+            2048,
+            2560,
```

Comment:
These are just temporary, right?

Reply:
We need to keep both resolutions. For now we'll use them like this.